transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoop.java

```diff
@@ -390,8 +390,8 @@ protected void run() {
                     //increase the size of the array as we needed the whole space for the events
                     events.increase();
                 }
-            } catch (Throwable t) {
```

Comment:
Use `throwable`.

Reply:
Can we have this change as a separate PR but for all EventLoop implementations ?

Reply:
Sure, created https://github.com/netty/netty/pull/10694